src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -20,6 +22,35 @@ public static List<Provider> ToProviders(string providers)
                 new List<Provider>() : providers.Split(',').Select(ToProvider).ToList();
         }
 
+        private static EventLevel GetEventLevel(string token)
+        {
+            if (Int32.TryParse(token, out int level) && level >= 0)
+            {
+                return level > (int)EventLevel.Verbose ? EventLevel.Verbose : (EventLevel)level;
```

Comment:
Per the old location's comment, do we want to consider throwing for levels beyond `5`?

Reply:
Hmm... I'm not sure if we should throw here. dotnet-trace already shows the current level and the number associated for each provider enabled, so I'm thinking we should keep the current behavior.